src/NodeParser/IntrinsicNodeParser.ts

```diff
@@ -6,33 +6,51 @@ import type { BaseType } from "../Type/BaseType.js";
 import { LiteralType } from "../Type/LiteralType.js";
 import { UnionType } from "../Type/UnionType.js";
 import { extractLiterals } from "../Utils/extractLiterals.js";
+import { isExtendsType } from "../Utils/isExtendsType.js";
+import { IntrinsicType } from "../Type/IntrinsicType.js";
+import { StringType } from "../Type/StringType.js";
 
-export const intrinsicMethods: Record<string, ((v: string) => string) | undefined> = {
+export const intrinsicMethods = {
```

Comment:
```suggestion
export const intrinsicMethods: Record<string, (str: string) => string> = {
```

Reply:
No need to use as const satisfies here: Type it directly and avoid the method access, using `intrinsicMethods[methodName]` will also be easier to read.